### PR TITLE
Model structure generic

### PR DESCRIPTION
**Pull request by gislenius**

### CODE REVIEW

**Comment by gislenius** [comment]
http://edinet-dev.28.io/v1/_queries/public/api/modelstructure-for-component.jq?_method=POST&token=foobar&aid=S10038VD&format=json&role=http%3A%2F%2Fdisclosure.edinet-fsa.go.jp%2Frole%2Fjpcrp%2Frol_CoverPage&profile-name=japan

File: queries/modules/io/28/modules/xbrl/components.jq [hunk 1]
```diff
@@ -1067,3 +1067,54 @@ declare function components:validation-errors(
...
+    as object*
+{
+  for $xbrl-concept in $xbrl-concepts
+  order by $xbrl-concept.Order
+  let $main-object := ($component.Concepts[])[$$.Name eq $xbrl-concept.Name]
+  let $kind := $main-object.Kind
+  return {|
+    project($xbrl-concept, ("Name", "Label", "Order")),
+    {
+      "Kind" : $main-object.Kind,
```
Comment (dknochen) [inline queries/modules/io/28/modules/xbrl/components.jq]: $main-object.Kind -> $kind

**Comment by dknochen** [comment]
very good improvements! Also for the SEC part of it.

Could we add a very simple test for the edinet config?

thank you.

**Comment by gislenius** [comment]
@dknochen good point, I added the test.

**Pull request merged**